applicationset/services/pull_request/gitea.go

```diff
@@ -45,7 +45,7 @@ func NewGiteaService(ctx context.Context, token, url, owner, repo string, insecu
 	}, nil
 }
 
-func (g *GiteaService) List(ctx context.Context) ([]*PullRequest, error) {
+func (g *GiteaService) List(_ context.Context) ([]*PullRequest, error) {
```

Comment:
I think we should probably actually add a call to `g.client.SetContext` instead of dropping the param.